content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -252,7 +251,7 @@ $ curl \
   --header "Content-Type: application/vnd.api+json" \
   --request PATCH \
   --data @payload.json \
-  https://app.terraform.io/api/v2/compound-workspaces/ws-erEAnPmgtm5mJr77
+  https://app.terraform.io/api/v2/organizations/my-organization/workspaces/my-workspace-2
```

Comment:
Is there a reason this doesn't use the same demo name as above?

Reply:
There are several workspace names used including my-workspace, my-workspace-1, my-workspace-2, and workspace-demo.  I did not want to make more changes than necessary.  I did make sure names were consistent within single examples.  Note that we need more than one name for the List Workspaces API.

Reply:
I suspect workspace-demo is valid as separate name because it is for workspace without VCS and should be distinct from others.

Reply:
I'm going to reduce to workspace-1 and workspace-2 with first for without VCS and second with.